pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -467,7 +476,15 @@ def _update_task_incoming(
         return db_document
 
     def _set_logs(self, payloads: dict, start_time: str) -> Dict:
-        """Set up the logs for the incoming request."""
+        """Set up the logs for the incoming request.
+
+        Args:
+            payloads: A dictionary containing the payload for the update.
```

Comment:
It should be `payload` (singular) not `payloads`.

Reply:
Done